ddtrace/ext/git.py

```diff
@@ -76,19 +77,46 @@ def is_ref_a_tag(ref):
     return "tags/" in ref if ref else False
 
 
-def _git_subprocess_cmd(cmd, cwd=None, std_in=None):
-    # type: (Union[str, list[str]], Optional[str], Optional[bytes]) -> str
-    """Helper for invoking the git CLI binary."""
+def _git_subprocess_cmd_with_details(cmd, cwd=None, std_in=None):
+    # type: (Union[str, list[str]], Optional[str], Optional[bytes]) -> Tuple[str, str, float, int]
+    """Helper for invoking the git CLI binary
+
+    Returns a tuple containing:
+        - a str representation of stdout
+        - a str representation of stderr
+        - the time it took to execute the command, in milliseconds
+        - the exit code
+    """
     if isinstance(cmd, six.string_types):
         git_cmd = cmd.split(" ")
     else:
         git_cmd = cmd  # type: list[str]  # type: ignore[no-redef]
     git_cmd.insert(0, "git")
+
+    stopwatch = StopWatch()
```

Comment:
Is this not a context manager?
```suggestion
    with StopWatch() as sw:
```